feature/stamps/src/main/java/io/github/droidkaigi/confsched2023/stamps/section/StampsSheet.kt

```diff
@@ -0,0 +1,38 @@
+package io.github.droidkaigi.confsched2023.stamps.section
+
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import io.github.droidkaigi.confsched2023.stamps.component.Stamps
+import io.github.droidkaigi.confsched2023.stamps.component.StampsDetail
+
+data class StampsSheetUiState(
+    val isStampA: Boolean,
```

Comment:
How about naming it `hasStampA`? Because UiState is not StampA 👀

Reply:
@takahirom 
Certainly, the naming seems to be good!